pod-operation/src/state_machine.rs

```diff
@@ -9,7 +10,8 @@ use tracing::info;
 
 // use crate::components::signal_light::SignalLight;
 
-const TICK_INTERVAL: Duration = Duration::from_millis(500);
+const TICK_INTERVAL: Duration = Duration::from_millis(10);
+const ENCODER_LIMIT: f32 = 37.0;
```

Comment:
Suggestion: the term "limit" isn't quite accurate as the encoder itself is not limited by such a value. Consider renaming to something like `COMPLETION_DISTANCE` or `STOP_THRESHOLD`.
Chore: for clarity, could we include an inline comment on the units of this value (i.e. feet, meters, etc.).